src/lib/xmlparse.rs

```diff
@@ -3908,7 +3914,7 @@ impl<'scf> XML_ParserStruct<'scf> {
                                         *fresh3 = *fresh2
                                     }
                                     prefix = tag.name.prefix as *mut XML_Char;
-                                    if self.m_ns_triplets as c_int != 0 && !prefix.is_null() {
```

Comment:
Similar to the style change comment above: this one isn't just style, it could be an actual semantic change but still should have its own commit.